src/beanahead/utils.py

```diff
@@ -67,6 +76,26 @@
 
 LEDGER_FILE_KEYS = ["x", "rx"]
 
+RootAccountsContext = {}  # global context
+
+
+def set_root_accounts_context(path_ledger: str) -> dict[str]:
```

Comment:
Would you mind adding a simple test to verify that all's working as expected when users are using non-default category names?